composer.json

```diff
@@ -17,6 +17,7 @@
     "require": {
         "php": "^7.4|^8.0",
         "drewroberts/blog": "^4.5.2",
+        "sixlive/nova-text-copy-field": "^1.5",
```

Comment:
Will need to update this simliary to https://github.com/drewroberts/blog/pull/88#discussion_r615536359

Reply:
Thank you. Can you create an issue on the blog package for it?